andela/ah-aquaman-frontend
# #164069196 list users bookmarks

**What does this PR do**
This PR Enables users to there bookmarks

**Description of tasks to be completed**

- List the users bookmarks
- ensure user can click on bookmark and be rerouted

**How can this be tested??**

- clone the repo
- checkout ft-list-bookmarks-profiles-164069196
- npm install && npm start
- login with your account credentials
- visit /profile
- you will see a list of your bookmarks

**Relevant pivotal tracker stories**

- [#164069196](https://www.pivotaltracker.com/story/show/164069196)

**Screenshots**
<img width="1394" alt="Screenshot 2019-04-26 at 15 45 09" src="https://user-images.githubusercontent.com/40163337/56808583-83caab00-683a-11e9-8b45-b305cbf7f5d0.png">
